### PR TITLE
test: add circle ci

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - run:
+          name: Execute main binary
+          command: go run cmd/scw/main.go -h
+
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - test
```
Comment (kindermoumoute) [inline .circleci/config.yml]: nit: EOL

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/golang:1.13
```
Comment (QuentinBrosse) [inline .circleci/config.yml]: Add go1.12?

Comment (jerome-quere) [reply .circleci/config.yml]: We don't need as we are building a binary. It's not a library

Comment (QuentinBrosse) [reply .circleci/config.yml]: Nevertheless useful for contributors. Also, some package manager need a client side compilation.

**Pull request merged**